README.md

```diff
@@ -12,10 +12,8 @@ Create a new Cordova Project
     
 Install the plugin
 
-    $ cd /to/project/folder/
-
-    $ cordova plugin add https://github.com/RADAR-CNS/RADAR-Cordova-OpenSmile-Plugin.git
-
+    $ cd hello
+    $ cordova plugin add https://github.com/nithinth7/plugin-opensmile.git
```

Comment:
Should this be `RADAR-base/RADAR-Cordova-OpenSmile-Plugin` ?

Reply:
you are right. corrected now.